aspnetcore/security/authentication/identity-configuration.md

```diff
@@ -94,17 +118,35 @@ Under `ConfigureServices` in the `Startup` class, you can configure the applicat
-| `ReturnUrlParameter`  | Determines the name of the query string parameter which is appended by the middleware when a 401 Unauthorized status code is changed to a 302 redirect onto the login path.  | ReturnUrl |
-| `AuthenticationScheme`  | This is only relevant for ASP.NET Core 1.x. The logical name for a particular authentication scheme. |  |
-| `AutomaticAuthenticate`  | This flag is only relevant for ASP.NET Core 1.x. When true, cookie authentication should run on every request and attempt to validate and reconstruct any serialized principal it created.  |  |
+[CookieAuthenticationOptions](/dotnet/api/microsoft.aspnetcore.authentication.cookies.cookieauthenticationoptions) has the following properties:
+
+| Property | Description |
+| -------- | ----------- |
+| [AccessDeniedPath](/dotnet/api/microsoft.aspnetcore.authentication.cookies.cookieauthenticationoptions.accessdeniedpath) | Informs the handler that it should change an outgoing *403 Forbidden* status code into a *302 Redirect* onto the given path.<br><br>The default value is `/Account/AccessDenied`. |
+| [AuthenticationScheme](/dotnet/api/microsoft.aspnetcore.builder.authenticationoptions.authenticationscheme) | Only applies to ASP.NET Core 1.x.<br><br> The logical name for a particular authentication scheme. |
+| [AutomaticAuthenticate](/dotnet/api/microsoft.aspnetcore.builder.authenticationoptions.automaticauthenticate) | Only applies to ASP.NET Core 1.x.<br><br> When true, cookie authentication should run on every request and attempt to validate and reconstruct any serialized principal it created. |
+| [AutomaticChallenge](/dotnet/api/microsoft.aspnetcore.builder.authenticationoptions.automaticchallenge) | Only applies to ASP.NET Core 1.x.<br><br> If true, the authentication middleware handles automatic challenges. If false, the authentication middleware only alters responses when explicitly indicated by the `AuthenticationScheme`. |
+| [ClaimsIssuer](/dotnet/api/microsoft.aspnetcore.authentication.authenticationschemeoptions.claimsissuer) | Gets or sets the issuer that should be used for any claims that are created (inherited from [AuthenticationSchemeOptions](/dotnet/api/microsoft.aspnetcore.authentication.authenticationschemeoptions)). |
+| [Cookie.Domain](/dotnet/api/microsoft.aspnetcore.http.cookiebuilder.domain) | The domain to associate the cookie with. |
+| [Cookie.Expiration](/dotnet/api/microsoft.aspnetcore.http.cookiebuilder.expiration) | Gets or sets the lifespan of a cookie. |
+| [Cookie.HttpOnly](/dotnet/api/microsoft.aspnetcore.http.cookiebuilder.httponly) | Indicates whether a cookie is accessible by client-side script.<br><br>The default value is `true`. |
+| [Cookie.Name](/dotnet/api/microsoft.aspnetcore.http.cookiebuilder.name) | The name of the cookie.<br><br>The default value is `.AspNetCore.Cookies`. |
+| [Cookie.Path](/dotnet/api/microsoft.aspnetcore.http.cookiebuilder.path) | The cookie path. |  |
```

Comment:
Looks like there's an extra pipe at the end of this line.

Reply:
Great :eye: